ksql-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java

```diff
@@ -639,4 +651,21 @@ public void shouldSetShowDescriptionsForShowQueriesDescriptions() {
     ListQueries listQueries = (ListQueries)statement;
     Assert.assertThat(listQueries.getShowExtended(), is(true));
   }
+  
+  private void assertQuerySucceeds(Statement statement) {
+    assertTrue(statement instanceof Query);
+  }
+
+  private void assertQueryFails(String sql, String exceptionMessage) {
+    try {
+      KSQL_PARSER.buildAst(sql, metaStore).get(0);
+      fail(format("Expected query: %s to fail with message: %s", sql, exceptionMessage));
+    } catch (ParseFailedException exp) {
+      if(!exp.getMessage().equals(exceptionMessage)) {
+        fail(format("Expected exception message to match %s for query: %s", exceptionMessage, sql));
+      }
+    } catch (RuntimeException exp) {
```

Comment:
Don't need this

Reply:
could you please elaborate why it's not needed?